README.md

```diff
@@ -761,6 +772,39 @@ Configure Slack to send notifications to specific channels. Provide your Slack t
 
 </details>
 
+
+<details>
+<summary>OpenSearch Integration</summary>
+
+The OpenSearch toolset (`opensearch`) allows Holmes to consult an opensearch cluster for its health, settings and shards information.
+The toolset supports multiple opensearch or elasticsearch clusters that are configured by editing Holmes' configuration file (or in cluster to the configuration secret):
+
+```                                                                                 
+opensearch_clusters:
+  - hosts:
+      - https://my_elasticsearch.us-central1.gcp.cloud.es.io:443
+    headers:
+      Authorization: "ApiKey <your_API_key>"
+# or
```

Comment:
The # are intended?

Reply:
yea, incase someone copies everything, to keep codeblock still functional